packages/web/src/components/SignInForm/styles.scss

```diff
@@ -9,4 +9,12 @@ form {
   .register {
     margin-top: 16px;
   }
-}
\ No newline at end of file
+}
+
+.spinner-container {
```

Comment:
this part  duplicates in packages/web/src/components/SignUpForm/styles.scss
I'd suggest move .spinner-container to common styles

Reply:
[Fixed](https://github.com/dubbha/croton/pull/38/commits/5836f4f37ab47be4d8fe6ea63ab42c9bf4ade443)